lib/archethic/utils.ex

```diff
@@ -913,4 +916,62 @@ defmodule Archethic.Utils do
     |> Stream.take_while(&(NaiveDateTime.compare(&1, end_of_month_datetime) in [:lt]))
     |> Enum.count()
   end
+
+  @doc """
+  get token properties based on the genesis address and the transaction
+  """
+  @spec get_token_properties(binary(), Transaction.t()) :: {:ok, map()} | {:error, atom()}
```

Comment:
You may specify the possible atom return when an error occur, it will be easier to understand the possible result

Reply:
done